Packs/MicrosoftDefenderAdvancedThreatProtection/CorrelationRules/MicrosoftDefenderAdvancedThreatProtection_correlation-rule.yml

```diff
@@ -1,24 +1,22 @@
 alert_category: EXECUTION
 alert_description: This alert will trigger in an event where Malware was detected by Microsoft Defender for Endpoints.
 alert_fields:
-  actor_process_image_name:
-  actor_process_image_path: xdm.target.file.path
+  action_local_ip: xdm.target.ipv4
+  actor_effective_username: xdm.source.user.username
   actor_process_command_line: xdm.target.process.command_line
-  agent_device_domain: xdm.source.user.domain
+  actor_process_image_path: xdm.target.file.path
   actor_process_image_sha256: xdm.target.file.sha256
+  agent_device_domain: xdm.source.user.domain
   agent_hostname: xdm.target.host.fqdn
-  action_local_ip: xdm.target.ipv4
-  action_remote_ip:
-  action_remote_port:
-  actor_effective_username: xdm.source.user.username
+  eventid: xdm.event.id
 alert_name: Malware detected by Microsoft Defender for Endpoints
 crontab: '*/10 * * * *'
 dataset: alerts
-description: This alert will trigger in an event where Malware was detected by Microsoft Defender for Endpoints.
+description: Alert will trigger an event when Malware ks detected by Microsoft Defender for Endpoints.
```

Comment:
I think you have a typo here + I would align this with the alert_description field.
I would change this sentence grammatically, as well - you are missing some determiners and prepositions (an + in).
I would change to "An alert will trigger" or "This alert will trigger" and "in an event", etc.

Reply:
@OBavly Thank you!
Updated the description.